test/metrics.js

```diff
@@ -0,0 +1,90 @@
+/* global describe, it, before, beforeEach, after, afterEach */
+
+'use strict';
+
+var assert = require('assert');
+var moment = require('moment');
+var Model = require('../src/model');
+
+var Errors = require('../src/errors');
+var expect = require('chai').expect;
+var util = require('../src/util');
+
+
+
+var EventEmitter = require('events').EventEmitter;
+var chai = require('chai');
+var should = chai.should();
+
+var EventBus = require('../src/core/event/event-bus.js').EventEmitter;
+
+var Metrics = require('../src/metrics');
+Metrics.init(EventBus);
+
+//var metricsEventEmitter = new EventBus;
+//var metricsEventEmitter = new EventEmitter;
+
+var sinon = require('sinon');
```

Comment:
looks like sinon isn't being used, delete this and remove from package.json if it's still there